packages/p2p-v2/src/components/AdvertiserName/AdvertiserNameStats.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { useAdvertiserStats, useDevice } from '../../hooks';
+import { StarRating } from '../StarRating';
+import ThumbUpIcon from '../../public/ic-thumb-up.svg';
+import BlockedUserOutlineIcon from '../../public/ic-user-blocked-outline.svg';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import './AdvertiserNameStats.scss';
+
+/**
+ * This component is to show an advertiser's stats, in UI its commonly used under an advertiser's name
+ * Example:
+ * Joined 2d | Not rated yet | x x x x x (5 ratings)
+ *
+ * Use cases are to show this in My Profile and Advertiser page
+ */
+const AdvertiserNameStats = () => {
+    const { data: advertiserStats, isLoading } = useAdvertiserStats();
+    const { isMobile } = useDevice();
+
+    // TODO: Use Skeleton loader here
+    if (isLoading || !advertiserStats) return <h1>Loading...</h1>;
+
+    const { blocked_by_count, daysSinceJoined, rating_average, rating_count, recommended_average } = advertiserStats;
+
+    return (
+        <div className='p2p-v2-advertiser-name-stats'>
+            <div>
+                <Text color='less-prominent' size='sm'>
+                    Joined {daysSinceJoined}d
+                </Text>
+            </div>
+            {!rating_average && (
+                <div>
+                    <Text color='less-prominent' size='sm'>
+                        Not rated yet
+                    </Text>
+                </div>
+            )}
+            {rating_average && (
+                <>
```

Comment:
do we need this wrapper here?

Reply:
Noted, I can fix it in the next PR I'm working on right now, cuz I need this profile content for constructing the profile page